ase/Makefile

```diff
@@ -501,6 +501,11 @@ help: header
 
 ## Build ASE Software objects and shared library ##
 sw_build: $(SIM_QUARTUS_VERILOG) $(SIM_QUARTUS_VHDL)
+ifdef ASE_HSSI_PLUGIN_PATH
+	cd $(WORK) ; cp $(ASE_HSSI_PLUGIN_PATH) $(HSSI_PLUGIN_SO)
```

Comment:
If you make this a link instead of a copy then you could be able to recompile your plugin without having to rebuild the simulation. You will probably have to add rm $(HSSI_PLUGIN_SO) in case it's rebuilding.

Reply:
Done, using symbolic link instead of copy now.